crates/tools/Cargo.toml

```diff
@@ -42,4 +43,14 @@ hex = "0.4"
 tokio = { version = "1.15", features = ["full"] }
 jsonrpc-core = "17"
 csv = "1.1.6"
+bech32 = "0.8.1"
 tracing-subscriber = { version = "0.3.11", default-features = false, features = ["tracing-log"] }
+parking_lot = "0.12"
+lru = "0.8"
+dashmap = "5.4"
+
+[dev-dependencies]
+ckb-chain-spec = "0.105.1"
+ckb-script = "0.105.1"
+httpmock = "0.6"
```

Comment:
This pulls in a lot of weight...

Reply:
Actually, we can remove the `httpmock`, it only used in the `ckb-sdk` tests...